UPGRADE-5.0.md

```diff
@@ -26,10 +29,7 @@ Removed stuff:
 * `FOS\ElasticaBundle\Doctrine\ORM\Provider` class was removed.
 * `FOS\ElasticaBundle\Doctrine\PHPCR\SliceFetcher` class was removed.
 * `FOS\ElasticaBundle\Doctrine\PHPCR\Provider` class was removed.
-* `FOS\ElasticaBundle\Propel\Provider` class was removed.
```

Comment:
This has to be reverted

Reply:
I added that propel support was removed. This class is only a subset of propel classes. So I don't see the value in only mentioning this class.